_episodes/04-drawing-bitwise.md

```diff
@@ -414,39 +371,34 @@ The resulting masked image should look like this:
 > >  * Python program to mask out everything but the wells 
 > >  * in a standardized scanned 96-well plate image.
 > > '''
-> > import cv2
 > > import numpy as np
+> > import skimage
+> > from skimage.viewer import ImageViewer
 > > import sys
 > > 
-> > # read in original image and open the centers file
-> > originalImage = cv2.imread(filename = sys.argv[1])
-> > centerFile = open(file = 'centers.txt')
+> > # read in original image
+> > image = skimage.io.imread(sys.argv[1])
```

Comment:
I would not introduce command line arguments for this, especially since the `centers.txt` file is still hardcoded.